content/ai/concepts/search-algorithms/terms/depth-first-search/depth-first-search.md

```diff
@@ -0,0 +1,87 @@
+---
+Title: 'Depth-First Search'
+Description: 'A shortest-path and traversing algorithm for unweighted graphs.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Graphs'
+  - 'Graph Search'
+  - 'Algorithms'
+CatalogContent:
+  - 'learn-python'
+  - 'paths/computer-science'
+---
+
+*Depth-first search (DFS)* is a traversing algorithm for unweighted graphs. This is a foundational algorithm in [graph](https://www.codecademy.com/resources/docs/general/graph) theory from which many other algorithms start.
```

Comment:
```suggestion
**Depth-first search (DFS)** is a traversing algorithm for unweighted graphs. This is a foundational algorithm in [graph](https://www.codecademy.com/resources/docs/general/graph) theory from which many other algorithms start.
```

Reply:
It looks like one of our existing entries, breadth-first search, was used as a starting point for this submission. There isn't anything wrong with using existing content as a reference, or inspiration, but a new submission cannot directly copy content from another. There are several instances in this entry where sentences from the previous entry have not been edited or rephrased for this submission. Please review and revise this entry to ensure that all of the content is original.

Reply:
> It looks like one of our existing entries, breadth-first search, was used as a starting point for this submission. There isn't anything wrong with using existing content as a reference, or inspiration, but a new submission cannot directly copy content from another. There are several instances in this entry where sentences from the previous entry have not been edited or rephrased for this submission. Please review and revise this entry to ensure that all of the content is original.

Done respected sir please review my PR.